src/components/categories.jsx

```diff
@@ -0,0 +1,7 @@
+const Categories = () => (
+  <>
+    <h1>Under construction⚠️</h1>
```

Comment:
- [ ] Great job. but according to the guidlines, the categories page should display a button that checks for a status. 
you can add a `<Button>` with the text "Check status"